foo-bar.php

```diff
@@ -44,7 +44,7 @@
  * Admin notice for incompatible versions of PHP.
  */
 function _foo_bar_php_version_error() {
-	printf( '<div class="error"><p>%s</p></div>', esc_html( _foo_bar_php_version_text() ) );
+	printf( '<div class="error"><p>%s</p></div>', _foo_bar_php_version_text() );
```

Comment:
Could we keep the `esc_html()` sanitiser during the output here? Currently it is being removed from all paces. 

The best practice would be to sanitize late when we know the type of output we're serving. Technically, the return value of `_foo_bar_php_version_text()` could be used for things like REST API errors, JS callback errors which would each require a different santiser.